generator.yml

```diff
@@ -91,6 +91,16 @@ allconfigs:
   - &x86_64_nx_am {<< : *x86_64_mn_am}
   - &x86_64_nx_an {<< : *x86_64_mn_an}
   - &x86_64_nx_ay {<< : *x86_64_mn_ay}
+  #                                       https://git.kernel.org/linus/28187dc8ebd938d574edfc6d9e0f9c51c21ff3f4
```

Comment:
I'd rather backport this to stable than have to have separate configs in our CI disabling BE.

Reply:
Fair, I will request the backport then enable these targets when it is merged.

Reply:
https://lore.kernel.org/r/20210205041351.GA2494386@localhost/